app/error.tsx

```
@@ -0,0 +1,73 @@
+
+  useEffect(() => {
+    try {
+      // Log the error to Sentry with route information
+      // Temporarily commented out until Sentry is fully configured
+      // Sentry.captureException(error, {
+      //   tags: { route: pathname || 'unknown' }
+      // });
+    } catch (sentryError) {
+      // Fallback if Sentry has issues
+      console.error('Failed to report to Sentry:', sentryError);
+    }
```

Biome (1.9.4)

[error] This code is unreachable



(lint/correctness/noUnreachable)